doc/qbk/2.0.quicklook.qbk

```diff
@@ -22,23 +22,37 @@ scope.
 Alternatively, individual headers may be included to obtain the declarations
  for specific types.
 
-To link your program you will need to link with a built library.
-Alternatively you can use the header-only configuration simply
-by including this header file in ['exactly one] of your new or
-existing source files:
+[heading Linking]
+
+You will need to link your program with the Boost.URL built library. You must install binaries
+in a location that can be found by your linker.
+
+If you followed the [@http://www.boost.org/doc/libs/release/more/getting_started/index.html Boost Getting Started]
+instructions, that's already been done for you.
+
+[heading Header-only]
+
+Alternatively Boost.URL can also be used as a header-only library. Simply include the Boost header directory in your
+compiler include path and use the header-only configuration by including this header file in ['exactly one] of your
+source files:
```

Comment:
LOL you said header and include three times in one sentence.

Reply:
Fixed.